### ISSUE TITLE
alma.get_data_info (following https://astroquery.readthedocs.io/en/latest/alma/alma.html) --> non-string array

### ISSUE THREAD

**Issue by PhilippeSalome** [astropy/astroquery#2015]
It seems that get_data_info gives a TypeError. 

```python
In [71]: astroquery.__version__
Out[71]: '0.4.2.dev6635'

In [67]: alma.get_data_info('uid://A001/X879/Xa3a')
WARNING: AstropyDeprecationWarning: Using the table property is deprecated. Please use se to_table() instead. [pyvo.dal.query]
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-67-a3f3cd1e8a27> in <module>
----> 1 alma.get_data_info('uid://A001/X879/Xa3a')

~/anaconda2/lib/python3.7/site-packages/astroquery-0.4.2.dev6635-py3.7.egg/astroquery/alma/core.py in get_data_info(self, uids, expand_tarfiles, with_auxiliary, with_rawdata)
    635         else:
    636             result = result[np.logical_or(np.core.defchararray.find(
--> 637                 result['semantics'], DATALINK_SEMANTICS) == -1,
    638                 result['content_type'] != DATALINK_FILE_TYPE)]
    639 

<__array_function__ internals> in find(*args, **kwargs)

~/anaconda2/lib/python3.7/site-packages/numpy/core/defchararray.py in find(a, sub, start, end)
    715     """
    716     return _vec_string(
--> 717         a, int_, 'find', [sub, start] + _clean_args(end))
    718 
    719 

TypeError: string operation on non-string array
```

**Comment by keflavich** [comment]
cc @andamian

**Comment by andamian** [comment]
Looking into it.

**Comment by aida-ahmadi** [comment]
Hi all, I'm waiting on this issue, wondering if it's been fixed and can be merged? Thank you!

**Comment by keflavich** [comment]
@aida-ahmadi it's merged, though your comment should have been on #2022

**Comment by keflavich** [comment]
closed by #2022
